src/Resources/app/administration/src/module/sw-cms/elements/twitter-buttons/config/index.js

```diff
@@ -0,0 +1,94 @@
+import template from './sw-cms-el-config-twitter-buttons.html.twig';
+
+const { Component, Mixin } = Shopware;
+
+Component.register('sw-cms-el-config-twitter-buttons', {
+    template,
+
+    inject: [
+        'configDataProviderService'
+    ],
+
+    mixins: [
+        Mixin.getByName('cms-element')
+    ],
+
+    watch: {
+        'element.config.twitterButtons': {
+            handler() {
+                this.configDataProviderService.setSyncData(this.$data.buttonOptions);
+            },
+            deep: true
+        }
+    },
+
+    data() {
+        return {
+            userId: '',
+            handle: '',
+            buttonType: '',
+            buttonOptions: [
+                {
+                    id: 'share',
+                    name: this.$tc('sw-cms.elements.twitterButtons.share.label'),
+                    href: 'https://twitter.com/share?ref_src=twsrc%5Etfw',
+                    class: 'twitter-share-button',
+                    message: 'Tweet',
+                },
+                {
+                    id: 'follow',
+                    name: this.$tc('sw-cms.elements.twitterButtons.follow.label'),
+                    href: this.replaceWithDataAttr('https://twitter.com/__ATTR__?ref_src=twsrc%5Etfw', 'handle'),
+                    class: 'twitter-follow-button',
+                    message: this.replaceWithDataAttr('Follow @__ATTR__', 'handle'),
+                },
+                {
+                    id: 'mention',
+                    name: this.$tc('sw-cms.elements.twitterButtons.mention.label'),
+                    href: this.replaceWithDataAttr('https://twitter.com/intent/tweet?screen_name=__ATTR__&ref_src=twsrc%5Etfw', 'handle'),
```

Comment:
We should not add **_?ref_src_** here.
It is just for tracking. Or am I wrong?

Reply:
I don't really know what you mean. Where do you see `?ref_src`.
The thing that i am seeing is that the href for this button type is missing altogether.

Reply:
After __ATTR__

Reply:
The url is taken from the example given by https://publish.twitter.com/#. I will look into it if it can be removed